ipfsconn/ipfshttp/ipfshttp_test.go

```diff
@@ -376,3 +376,26 @@ func TestConfigKey(t *testing.T) {
 		t.Error("should not work with a bad path")
 	}
 }
+
+func TestRepoGC(t *testing.T) {
+	ctx := context.Background()
+	ipfs, mock := testIPFSConnector(t)
+	defer mock.Close()
+	defer ipfs.Shutdown(ctx)
+
+	res, err := ipfs.RepoGC(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if res.Error != "" {
+		t.Errorf("expected error to be empty: %s", res.Error)
+	}
+
+	if res.Keys[0].Key.Equals(test.Cid1) {
+		t.Errorf("expected different cid, expected: %s, found: %s\n", test.Cid1, res.Keys[0].Key.String())
```

Comment:
`.String()` is not needed. Also, check that length of `res.Keys` is more than 0 to avoid panics.